srcpkgs/gcn64tools/template

```diff
@@ -0,0 +1,31 @@
+# Template file for 'gcn64tools'
+pkgname=gcn64tools
+_longname=raphnet-tech_adapter_manager
+version=2.1.19
+revision=1
+wrksrc="${_longname}-${version}"
+build_style=gnu-makefile
+make_use_env=true
+make_build_args="-C src"
```

Comment:
Not `buld_wrksrc`?

Reply:
The tarball has everything under that wrksrc directory, and with only build_wrksrc it can't find the sources. I couldn't get it to compile at all when wrksrc wasn't set.

Reply:
Should be wrksrc and build_wrksrc